pkg/scheduler/framework/plugins/requestedtocapacityratio/requested_to_capacity_ratio_test.go

```diff
@@ -0,0 +1,178 @@
+		expectedPriorities framework.NodeScoreList
+	}
+
+	tests := []test{
+		{
+			name:      "nothing scheduled, nothing requested (default - least requested nodes have priority)",
+			requested: resources{0, 0},
+			nodes: map[string]nodeResources{
+				"node1": {
+					capacity: resources{4000, 10000},
+					used:     resources{0, 0},
+				},
+				"node2": {
+					capacity: resources{4000, 10000},
+					used:     resources{0, 0},
+				},
+			},
+			expectedPriorities: []framework.NodeScore{{Name: "node1", Score: 100}, {Name: "node2", Score: 100}},
+		},
+		{
+			name:      "nothing scheduled, resources requested, differently sized machines (default - least requested nodes have priority)",
+			requested: resources{3000, 5000},
+			nodes: map[string]nodeResources{
+				"node1": {
+					capacity: resources{4000, 10000},
+					used:     resources{0, 0},
+				},
+				"node2": {
+					capacity: resources{6000, 10000},
+					used:     resources{0, 0},
+				},
+			},
+			expectedPriorities: []framework.NodeScore{{Name: "node1", Score: 38}, {Name: "node2", Score: 50}},
+		},
+		{
+			name:      "no resources requested, pods scheduled with resources (default - least requested nodes have priority)",
+			requested: resources{0, 0},
+			nodes: map[string]nodeResources{
+				"node1": {
+					capacity: resources{4000, 10000},
+					used:     resources{3000, 5000},
+				},
+				"node2": {
+					capacity: resources{6000, 10000},
+					used:     resources{3000, 5000},
+				},
+			},
+			expectedPriorities: []framework.NodeScore{{Name: "node1", Score: 38}, {Name: "node2", Score: 50}},
+		},
+	}
+
+	buildResourcesPod := func(node string, requestedResources resources) *v1.Pod {
+		return &v1.Pod{
+			Spec: v1.PodSpec{
+				NodeName: node,
+				Containers: []v1.Container{
+					{
+						Resources: v1.ResourceRequirements{
+							Requests: v1.ResourceList{
+								v1.ResourceCPU:    *resource.NewMilliQuantity(requestedResources.cpu, resource.DecimalSI),
+								v1.ResourceMemory: *resource.NewQuantity(requestedResources.mem, resource.DecimalSI),
+							},
+						},
+					},
+				},
+			},
+		}
+	}
+
+	fh, _ := framework.NewFramework(nil, nil, nil)
+	args := &runtime.Unknown{Raw: []byte(`{"FunctionShape" : [{"Utilization" : 0, "Score" : 100}, {"Utilization" : 100, "Score" : 0}], "ResourceToWeightMap" : {"memory" : 1, "cpu" : 1}}`)}
+	p, _ := New(args, fh)
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			state := framework.NewCycleState()
+
+			var nodeNames []string
+			for nodeName := range test.nodes {
+				nodeNames = append(nodeNames, nodeName)
+			}
+			sort.Strings(nodeNames)
+
+			var nodes []*v1.Node
+			for _, nodeName := range nodeNames {
+				node := test.nodes[nodeName]
+				nodes = append(nodes, makeNode(nodeName, node.capacity.cpu, node.capacity.mem))
+			}
+
+			var scheduledPods []*v1.Pod
+			for name, node := range test.nodes {
+				scheduledPods = append(scheduledPods,
+					buildResourcesPod(name, node.used))
+			}
```

Comment:
instead of this, create a []*v1.Pod in the test struct and use ```makePod``` to instantiate it using the values currently placed in ```used```.